src/full/Agda/TypeChecking/Serialise/Instances/Errors.hs

```diff
@@ -258,6 +258,15 @@ instance EmbPrj PragmaOptions where
       valuN PragmaOptions a b c d e f g h i j k l m n o p q r s t u v w x y z aa bb cc dd ee ff gg hh ii jj kk ll mm nn oo pp qq rr ss tt uu vv ww xx yy zz aaa bbb
     _ -> malformed
 
+instance EmbPrj UnicodeOrAscii where
```

Comment:
Note that you can skip implementing `EmbPrj` by hand if you make `UnicodeOrAscii` instance of `Enum` and `Bounded`.

Reply:
Ah! Thank you, that would be cleaner. I'll fix and push in a bit.

Reply:
Pushed. Compilation was tested locally so I'll merge to free up build cycles.